constrain/library/hot_water_reset.py

```diff
@@ -148,12 +148,12 @@ def verify(self):
                     (
                         self.df["temperature_water_hot"]
                         >= self.df["temperature_water_hot_setpoint_min"]
-                        + self.get_tolerance("temperature", "general")
+                        - self.get_tolerance("temperature", "general")
```

Comment:
Tolerance signs were incorrect and not caught by test since we didn't have a test for it.